### PR TITLE
Added the Balloon Selector field type

### PR DESCRIPTION
**Pull request by gergo85**

### CODE REVIEW

File: backend-forms.md [hunk 1]
```diff
@@ -363,6 +364,18 @@ Radio lists can also support a secondary description.
...
+
+`balloon-selector` - renders a list, where only one item can be selected at a time.
+
+    gender:
+        label: Gender
+        type: balloon-selector
+        options:
+            female: Female
+            male: Male
+
```
Comment (LukeTowers) [inline backend-forms.md]: Please add the extra information from the checkboxlist widget (excluding the bit about secondary descriptions, as `balloon-selector` doesn't actually currently support them.

```
Balloon selectors support three ways of defining the options, exactly like the [dropdown field type](#field-dropdown).
```

**Comment by LukeTowers** [comment]
Added a comment, once that's taken care of it should be good to merge :)

File: backend-forms.md [hunk 2]
```diff
@@ -363,6 +364,18 @@ Radio lists can also support a secondary description.
 
 Radio lists support three ways of defining the options, exactly like the [dropdown field type](#field-dropdown). For radio lists the method could return either the simple array: **key => value** or an array of arrays for providing the descriptions: **key => [label, description]**
 
+<a name="field-balloon"></a>
+### Balloon Selector
+
+`balloon-selector` - renders a list, where only one item can be selected at a time.
```
Comment (LukeTowers) [inline backend-forms.md]: I think that `renders a list of button options, where only one item can be selected at a time.` from #146 is a slightly better description as well, so if you could add that it'd be great.

**Comment by LukeTowers** [comment]
@gergo85 Any plans to make the requested changes?

**Comment by gergo85** [comment]
Is it correct?

**Comment by LukeTowers** [comment]
Works for me, thanks!

**Pull request merged**